src/storage/lock_manager/deadlock.rs

```diff
@@ -160,6 +167,26 @@ impl DetectTable {
     {
         self.wait_for_map.retain(|ts, _| !is_expired(*ts));
     }
+
+    /// The threshold of detect table size to trigger `active_expire`.
+    const ACTIVE_EXPIRE_THRESHOLD: usize = 100000;
```

Comment:
May there are few entries in `wait_for_map`, but too many stale entries in the inner maps?

Reply:
Actually, it's almost impossible to leave many entries in the inner map. Because if a transaction waits for many locks, which means the transaction may be across many regions, it will send many `CleanUp` requests( as many as involved regions) to the deadlock detector when the transaction is committed or rollbacked.

If it counts all entries in the table to trigger expiration, I'm afraid it will reduce performance.